src/client/components/BikeSummary.tsx

```diff
@@ -33,7 +33,8 @@ const BikeSummary: React.FC<Props> = ({
     getUrl();
   }, [loading]);
 
-  return distance ? (
+  // allow distance of 0
+  return distance !== undefined ? (
```

Comment:
fyi this check isn't needed, since distance wouldn't be true if it was undefined